exercises/luhn-from/example.rs

```diff
@@ -0,0 +1,62 @@
+pub struct Luhn {
+    digits: Vec<char>,
+}
+
+impl Luhn {
+    pub fn is_valid(&self) -> bool {
+        if self.digits.iter().any(|c| c.is_alphabetic()) || self.digits.iter().count() == 1 {
+            return false;
+        }
+
+        self.digits
+            .iter()
+            .filter_map(|c| c.to_digit(10))
+            .rev()
+            .enumerate()
+            .map(|(index, digit)| if index % 2 == 0 { digit } else { digit * 2 })
+            .map(|digit| if digit > 9 { digit - 9 } else { digit })
+            .sum::<u32>() % 10 == 0
+    }
+}
+
+impl From<String> for Luhn {
+    fn from(s: String) -> Self {
+        Luhn { digits: s.chars().collect() }
+    }
+}
+
+impl From<&'static str> for Luhn {
+    fn from(s: &'static str) -> Self {
```

Comment:
I understand it's only the example, but I'm not clear on why this would have to be `&'static str` specifically. What would happen if this were just `&str`? It seems unnecessarily restrictive to me to only accept `&'static str`.